### PR TITLE
Create recommended-material-for-applicants.md

### PR DESCRIPTION
**Pull request by sofer**
@bradreeder this is information for applicants that is linked to from the site, but it seems to me that it is relevant to the master-reference and could usefully go here (but not sure where).

### CODE REVIEW

File: recommended-material-for-applicants.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
+# Other recommended materials for applicants
+
+A collection of recommended study materials for people thinking of applying for the programme.
+
+[A Guide to setting up your developer environment](https://github.com/dwyl/dev-setup) by our good friends at dwyl is an excellent starting point for new developers.
```
Comment (nelsonic) [inline recommended-material-for-applicants.md]: ❤️ 
also: @shouston3 has https://github.com/shouston3/My-Config

File: recommended-material-for-applicants.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+[Khan Academy](https://www.khanacademy.org/computing) is an excellent resource for complete beginners. We love their introduction to Computer Science as well as their JavaScript-based programming course.
+
+[Codecademy](https://www.codecademy.com/) is another good resource for complete beginners. We recommend [HTML & CSS](https://www.codecademy.com/learn/web), [Make a Website](https://www.codecademy.com/en/skills/make-a-website), [JavaScript](https://www.codecademy.com/learn/javascript) and the [YouTube API](https://www.codecademy.com/en/tracks/youtube).
+
+[Eloquent Javascript](http://eloquentjavascript.net/) is a great self-contained introduction to Javascript. The first 4 chapters are relatively easy to follow. Subsequent chapters cover more advanced topics.
+
+[Internet History, Technology, and Security](https://www.coursera.org/learn/internet-history) on Coursera provides a nice bit of historical context.
+
+[Javascript: The Good Parts](http://bdcampbell.net/javascript/book/javascript_the_good_parts.pdf) is written by the godfather of Javascript, Douglas Crockford, but later chapters can be hard to follow.
```
Comment (nelsonic) [inline recommended-material-for-applicants.md]: Also: https://github.com/iteles/Javascript-the-Good-Parts-notes

**Comment by jsms90** [comment]
I'm unsure how helpful it is for us to provide a long list of resources that students _could_ use, with no particular guidance as to when in their journey it might be best to do so. One of the best things about joining a bootcamp, or even just going through FAC's pre-reqs is that, as a newbie, you feel less overwhelmed by the sheer quantity of resources out there. You feel like someone who has gone before you (or many people) have put time into carefully wading through the sea of noise for you and helping to show you the way.

I tried to make [our own](https://github.com/FAC9/notes/tree/master/resources-from-gitter) list during FAC9. But I can think of an absolute tonne of worthwhile things that we could add to this. Just having an unfiltered, ever-growing list isn't all that helpful in the end. I feel like FAC10 are coming to the same conclusions that I did, and it will be interesting to see if anything comes of [this proposal](https://github.com/foundersandcoders/master-reference/issues/255).

But I definitely do understand the desire not to lose track of which resources that our community enjoys.... I just wonder whether listing them in `master-reference` (especially under the title "recommended material for applicants") is really going to achieve what we want it to achieve? :thinking: 

@sofer what do you think?

**Comment by zurda** [comment]
@sofer, is this still in the making, or was this list already implemented (completely or partially) elsewhere? I'm happy to implement @nelsonic suggestions, but want to make sure this is still relevant before I do

**Comment by sofer** [comment]
Oh my gosh. This is an old one. @zurda I think we can probably close this. I agree with @jsms90 that *"a long list of resources that students could use"* is of uncertain value.
